modules/logger.lua

```diff
@@ -138,19 +139,18 @@ end
 ---@field source string source of the log. Usually a playerId or name of a resource.
 ---@field event string the action or 'event' being logged. Usually a verb describing what the name is doing. Example: SpawnVehicle
 ---@field message string the message attached to the log
----@field webhook? string Discord logs only. url of the webhook this log should send to
+---@field webhook? string Discord logs only. url of the webhook this log should send to. looks up via config
 ---@field color? string Discord logs only. what color the message should be
----@field tags? string[] Discord logs only. tags in discord. Example: ['@admin', '@everyone']
+---@field tags? string[] Discord logs only. tags in discord. Example: {'<@%roleid>', '@everyone'}
 
 ---Creates a log using either ox_lib logger, discord webhooks depending on convar
 ---@param log Log
 local function createLog(log)
     if log.webhook then
         ---@diagnostic disable-next-line: param-type-mismatch
         discordLog(log)
-    else
-        lib.logger(log.source, log.event, log.message)
     end
+    lib.logger(log.source, log.event, log.message) -- oxlib fails silently if logging isn't setup correctly.
```

Comment:
This assumes that someone will always want to log to ox_lib instead of some logs going to discord and other logs going to ox_lib. I think that's probably a good assumption, but wanted to call it out. No action required.

Reply:
Yep as discussed if you don't configure oxlib logging it just no-ops